WalletWasabi.Fluent/ViewModels/Wallets/Settings/CoinJoinSettingsViewModel.cs

```diff
@@ -90,23 +92,29 @@ private CoinJoinSettingsViewModel(IWalletModel walletModel)
 	protected override void OnNavigatedTo(bool isInHistory, CompositeDisposable disposables)
 	{
 		base.OnNavigatedTo(isInHistory, disposables);
+		Update();
```

Comment:
Since this viewmodel never actually gets navigated to, this will never be called.

Related: https://github.com/zkSNACKs/WalletWasabi/issues/6184